paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentMethodsUI.kt

```diff
@@ -48,7 +48,7 @@ internal fun PaymentMethodsUI(
     val state = rememberLazyListState()
 
     LaunchedEffect(selectedIndex) {
-        state.scrollToItem(selectedIndex, 0)
+        state.animateScrollToItem(selectedIndex, 0) // Klarana still didn't scroll
```

Comment:
We can omit the scroll offset here, as it defaults to 0.

```suggestion
        state.animateScrollToItem(selectedIndex)
```